scripts/yg-privacyenhanced.py

```diff
@@ -35,12 +35,10 @@
 class YieldGeneratorPrivacyEnhanced(YieldGeneratorBasic):
 
     def __init__(self, wallet, offerconfig):
-        self.txfee, self.cjfee_a, self.cjfee_r, self.ordertype, self.minsize \
-            = offerconfig
         super(YieldGeneratorPrivacyEnhanced, self).__init__(wallet, offerconfig)
 
     def create_my_orders(self):
-        mix_balance = self.wallet.get_balance_by_mixdepth()
+        mix_balance = self.get_available_mixdepths(verbose=False)
```

Comment:
This appears to be a bug? `YieldGenerator.get_available_mixdepths()` has no keyword argument `verbose`.

Reply:
Sorry I meant `YieldGeneratorBasic` not `YieldGenerator` but same comment.

Reply:
yes, indeed!

Reply:
Ah yes, I apparently missed that when I updated the PR for review comments.  Sorry - fix is in #410.